src/nodes/kolibri-in/kolibri-in.html/editor.html

```diff
@@ -7,7 +7,10 @@
       <label for="node-input-path"><i class="fa fa-tasks"></i> Data point path</label>
       <input type="text" id="node-input-path" placeholder="Path">
   </div>
-
+  <div class="form-row">
+    <label for="node-input-resume"><i class="fa fa-tasks"></i> Resume</label>
+    <input class="input-append-left" type="checkbox" id="node-input-resume" placeholder="false">
```

Comment:
check the checkbox indentation. It could be more to the left, aligned with the input fields.

Reply:
Please see solution